pkg/modeltranslator/model_translator.go

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+//nolint:all
```

Comment:
Why? What is the lint issue?

Reply:
its a hot fix until I find a solution for the lint issues